config/routes.rb

```diff
@@ -3,5 +3,8 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  root "users#new"
+  root 'users#index'
+  get 'users/index'
+  get 'users/sign_out'
+  resources :users, only: %i[index]
```

Comment:
Since you already has `resources :users, only: %i[index]`, I suggest removing the line 7 `get 'users/index'`.
Please remove also `only: %i[index`, because you'll need the other actions (like `:create`) as well.